signinwithapplebutton/src/main/res/values/styles.xml

```diff
@@ -1,25 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources xmlns:tools="http://schemas.android.com/tools">
-    <style name="SignInWithAppleButton">
+<resources>
+    <style name="SignInWithAppleButtonBase" />
+
+    <style name="SignInWithAppleButton" parent="SignInWithAppleButtonBase">
         <item name="android:background">@drawable/sign_in_with_apple_button_background_black</item>
         <item name="android:textColor">@color/sign_in_with_apple_button_text_white</item>
-        <item name="android:drawableLeft">@drawable/sign_in_with_apple_button_icon_white</item>
+        <item name="android:drawableLeft">@drawable/sign_in_with_apple_button_icon</item>
     </style>
 
     <style name="SignInWithAppleButton.Black">
         <!-- Inherits default settings. Black is the default if you don't specify a style. -->
     </style>
 
-    <style name="SignInWithAppleButton.White">
+    <style name="SignInWithAppleButtonBase.White"/>
+
+    <style name="SignInWithAppleButton.White" parent="SignInWithAppleButtonBase.White">
         <item name="android:background">@drawable/sign_in_with_apple_button_background_white</item>
         <item name="android:textColor">@color/sign_in_with_apple_button_text_black</item>
-        <item name="android:drawableLeft">@drawable/sign_in_with_apple_button_icon_black</item>
     </style>
 
-    <style name="SignInWithAppleButton.WhiteOutline">
+    <style name="SignInWithAppleButton.WhiteOutline" parent="SignInWithAppleButtonBase.White">
         <item name="android:background">@drawable/sign_in_with_apple_button_background_white_outline</item>
         <item name="android:textColor">@color/sign_in_with_apple_button_text_black</item>
```

Comment:
Can omit this term since its value is inherited from white

Reply:
It's inheriting from _Base.White_ (`SignInWithAppleButtonBase.White`). Could inharit from white instead and not include it here, was 50/50 on it. The point of the base themes is to be able to override on different api levels without duplicating ourselves. Could get rid of if we go minsdk 21.

Reply:
Ah got it. I missed that that whole layer exists.